### PR TITLE
zh-translation: /docs/setup/platform-setup/gardener/index.md

### PR DESCRIPTION
**Pull request by arunfung**
ref: https://github.com/servicemesher/istio-official-translation/issues/1294

### CODE REVIEW

File: content/zh/docs/setup/platform-setup/gardener/index.md [hunk 1]
```diff
@@ -1,76 +1,62 @@
...
 {{< text bash >}}
 $ kubectl --namespace shoot--my-project--my-cluster get secret kubecfg --output jsonpath={.data.kubeconfig} | base64 --decode > my-cluster.yaml
 {{< /text >}}
 
-This kubeconfig file has full administrators access to you cluster. For the rest
-of this guide be sure you have `export KUBECONFIG=my-cluster.yaml` set.
+此 kubeconfig 文件具有管理员对您群集的完全访问权限。对于本指南的其余部分，请确保已设置 `export KUBECONFIG=my-cluster.yaml`。
 
-## Cleaning up
+## 删除{#Cleaning-up}
```
Comment (rootsongjc) [inline content/zh/docs/setup/platform-setup/gardener/index.md]: 标题里的英文小写

Comment (arunfung) [reply content/zh/docs/setup/platform-setup/gardener/index.md]: > 标题里的英文小写

已修改

File: content/zh/docs/setup/platform-setup/gardener/index.md [hunk 2]
```diff
@@ -1,76 +1,62 @@
...
 $ kubectl apply --filename my-cluster.yaml
 {{< /text >}}
 
-The easier alternative is to create the cluster following the cluster creation
-wizard in the Gardener dashboard:
+更简单的替代方法是按照 Gardener 仪表板中的集群创建向导来创建集群：
 ![shoot creation](images/gardener_shoot_creation.png "shoot creation via the dashboard")
 
-### Configure `kubectl` for your cluster
+### 为集群配置 `kubectl`{#configure-`kubectl`-for-your-cluster}
```
Comment (rootsongjc) [inline content/zh/docs/setup/platform-setup/gardener/index.md]: 锚点里不能使用反引号。

**Pull request merged**